test/core/Rectangle.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-describe('PIXI.Rectangle', function ()
+describe('PIXI.Rectangle', () =>
 {
-    it('should create a new rectangle', function ()
+    it('should create a new rectangle', () =>
```

Comment:
@out-of-band probably my bad.

@bigtimebuddy it's only in the `describe` and `it` functions right?  `should`s and other inline stuff can still be fat arrows right?